frontend/src/static/js/main.js

```diff
@@ -118,6 +118,9 @@ window.app = new window.Vue({
       window.JSZip.loadAsync(evt.target.files[0])
           .then((zip) => {
             window.REPORT_ZIP = zip;
+          }, () => {
+            window.alert('Either the .zip file is corrupted, or you did not upload a .zip file that is generated '
```

Comment:
>did not upload a .zip file

Would the alert appear if the user did not upload a file? If not change the wording abit such that
`you have uploaded a zip file that is not generated...`